### PR TITLE
Update translations from crowdin

### PR DESCRIPTION
**Pull request by peppy**

### CODE REVIEW

File: resources/lang/vi/common.php [hunk 1]
```diff
@@ -73,11 +74,12 @@
         'minute_short_unit' => 'phút',
         'minutes' => ':count phút',
         'months' => ':count tháng',
-        'notifications' => '',
-        'plus_others' => '',
+        'notifications' => ':count_delimited thông báo|:count_delimited thông báo',
+        'plus_others' => '+ :count_delimited khác!+ :count_delimited khác!',
```
Comment (nanaya) [inline resources/lang/vi/common.php:78]: missing separator (`|`). I've updated it in crowdin so maybe run another update.

**Review by nanaya — CHANGES_REQUESTED** [review]

File: resources/lang/vi/common.php (hunk 1 above)
Comment (nanaya) [reply resources/lang/vi/common.php:78]: actually Vietnamese don't even need this as there's no change for plural things :thinking:

Comment (peppy) [reply resources/lang/vi/common.php:78]: Does that actually cause a runtime issue?

Comment (nanaya) [reply resources/lang/vi/common.php]: the extra versions are ignored. Although without the separator it'll just look weird...

**Review by nanaya — APPROVED** [review]

**Pull request merged**